tox.ini

```diff
@@ -73,15 +72,15 @@ commands = flake8-rst gensim/ docs/ {posargs}
 
 
 [testenv:compile]
-basepython = python2
+basepython = python3.7
 recreate = True
 
-deps = numpy==1.11.3
+deps = numpy==1.14.5
```

Comment:
maybe update numpy/scipy versions everywhere? (incl. deps above and `setup.py`)

This will allow installing numpy from pypi (without any `--trusted-host`)

btw, don't forget to mirror this change to `gensim-wheels` and `gensim-forge`)

Reply:
I moved this to install_wheels.py script